### DISCUSSION TITLE
Toggle between diff tool or no diff tool when test fails

### DISCUSSION THREAD

**Discussion by michelschep** [VerifyTests/Verify#826] [General]
Sometimes you just want to run all tests and not see the diff tool appear when the test fails.
Is there an (easy) way to toggle between opening a diff tool (when the test fails) or only outputting the test failed?

**Comment by SimonCropp** [comment]
u can do it in code or via an environment variable https://github.com/VerifyTests/DiffEngine#disable-in-code

which u could setup in a module init and then only trigger that based on a named solution configuration

**Reply by michelschep** [reply]
Thx Simon! That helps (and it works as well :-))

**Comment by SimonCropp** [comment]
if u used the resharper or rider extensions https://github.com/VerifyTests/Verify#snapshot-management

u could add a feature there to be able to toggle diff launching

**Reply by michelschep** [reply]
That's very handy extension indeed. Works as a charm. This week a presentation + demo about snapshot testing and guess which library I will use in the demo :-) So I will include how to use extension as well.

**Reply by SimonCropp** [reply]
whens your presentation? can i attend?

u should also cover diffenginetray
